docs/rst/introduction/tools/shapes_demo.rst

```diff
@@ -3,5 +3,136 @@
+    When a sample’s lifespan expires, it will be removed from publisher and subscriber histories (see Fast DDS LifespanQosPolicy Documentation).
+
+.. note::
+
+    Using Lifespan QoS will not have any visual effect.
+
+Subscribing to a Topic
+^^^^^^^^^^^^^^^^^^^^^^
+
+When the Subscriber button is pressed, a new window appear to allow the user to define the Shape (topic) and Quality of Service (QoS) for its subscription.
+The following image shows an example of the Subscribe menu.
+
+.. figure:: /rst/figures/intro/tools/shapes_demo/subscribe.png
+   :alt: Subscribe Window
+   :align: center
+
+This menu is highly similar to the Publication menu but the user cannot change the color and size of the Shape, and it has additional elements:
+
+*   Liveliness: This QoS policy is applied in the same way as in the publisher except for the Announcement Period, which does not apply for the Subscriber (see Fast DDS LivelinessQosPolicy Documentation).
+*   Time Based Filter: This value can be used by the user to specify the minimum amount of time (in milliseconds) that the subscriber wants between updates (see Fast DDS TimeBasedFilterQosPolicy Documentation).
+*   Content Based Filter: This filter draws a rectangle in the instances window.
+    Only the shapes that are included in this rectangle are accepted while the rest of them are ignored.
+    The user can dynamically resize and move this content filter.
+
+.. note::
+
+    Using Lifespan QoS will not have any visual effect.
+
+Participant configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The *ROS 2 Shapes Demo* application allows the user to define Participant policies.
+To see the Options window, please go to Options->Participant Configuration in the main bar.
+The following image shows the Options Menu.
+
+.. figure:: /rst/figures/intro/tools/shapes_demo/participant.png
+   :scale: 75 %
+   :alt: Options Window
+   :align: center
+
+*   Transport Protocol: You can select between UDP protocol, TCP LAN Server, TCP WAN Server or TCP Client, and Shared Memory protocol.
+
 .. note::
-    This page is under maintenance and will be updated soon.
+
+    This is under maintenance and will be updated soon.
+
+Preferences
+^^^^^^^^^^^
+The *ROS 2 Shapes Demo* application allows the user to define additional options.
+To see the Options window, please go to Options->Preferences in the main bar.
+The following image shows the Options Menu.
+
+.. figure:: /rst/figures/intro/tools/shapes_demo/preferences.png
+   :scale: 75 %
+   :alt: Options Window
+   :align: center
+
+The user can customize several aspects of Shapes Demo operation:
+
+* Update interval: This value changes the publication period for all the publishers.
+* Speed: This scroll bar allows the user to change how much the Shape moves between two write calls.
+
+Endpoints and Output tabs
+^^^^^^^^^^^^^^^^^^^^^^^^^
+.. note::
+
+    This is under maintenance and will be updated soon.
```

Comment:
Wouldn't be better to comment this as a todo?

Reply:
Same as before